Print output of predictions in simple estimator example for DistributionStrategy

jppgks opened the pull request:
Resolved the TODO (cc @anj-s) that asked to print meaningful results from the Estimator's predictions.
Collects all elements yielded by the generator in a list and prints that list instead of the generator object.
Results in the following output: `Prediction results: [{'logits': array([1.0162734], dtype=float32)}, ..., {'logits': array([1.0162734], dtype=float32)}]`.

Commit:
Print output of predictions in simple estimator example for DistributionStrategy

Resolved the TODO that asked to print meaningful results from the Estimator's predictions.
Collects all elements yielded by the generator in a list and prints that list instead of the generator object.
Results in the following output: `Prediction results: [{'logits': array([1.0162734], dtype=float32)}, ..., {'logits': array([1.0162734], dtype=float32)}]`.

tensorflowbutler commented:
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

anj-s commented on tensorflow/contrib/distribute/python/examples/simple_estimator_example.py:
```diff
@@ -78,9 +78,8 @@ def predict_input_fn():
-  # TODO(anjalsridhar): This returns a generator object, figure out how to get
-  # meaningful results here.
+  prediction_iterable = estimator.predict(input_fn=predict_input_fn)
+  predictions = [prediction_iterable.next() for _ in range(10)]
```
Thanks for adding this! Can we add a comment that predictions is a list of output dicts with key as 'logits' and value as an array of model outputs?

Commit:
Add comment clarifying items in `predictions` list

This way, it becomes clear what the result of the `print()` call will look like.

anj-s reviewed (approved):
Looks good. Thanks!

tensorflowbutler commented:
Nagging Assignee @protoget: It has been 33 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

tensorflowbutler commented:
Nagging Assignee @protoget: It has been 48 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

tensorflowbutler commented:
Nagging Assignee @protoget: It has been 63 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

tensorflowbutler commented:
Nagging Assignee @protoget: It has been 78 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

tensorflowbutler commented:
Nagging Assignee @protoget: It has been 93 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
